packages/interface/src/store/selectors.js

```diff
@@ -15,6 +20,7 @@ import { store as preferencesStore } from '@wordpress/preferences';
  */
 export const getActiveComplementaryArea = createRegistrySelector(
 	( select ) => ( state, scope ) => {
+		scope = normalizeComplementaryAreaScope( scope );
 		const isComplementaryAreaVisible = select( preferencesStore ).get(
 			scope,
```

Comment:
This will cause the preferences that are stored to be lost. E.g: with an update the items I had selected to be pinned or unpinned are reset.

Reply:
Let me see how to add a migration for that :)

Reply:
done in [6b5914e](https://github.com/WordPress/gutenberg/pull/60778/commits/6b5914e0f74424013a8b6490ab976200eea3f91b)

Reply:
I don't think it is a blocker if a migration is simple we can look into it, otherwise I guess is something acceptable in favor of better developer experience in the future.

Reply:
Yes, it's simple enough, we already did similar things for other preferences that were unified.

Reply:
oh so, this shows that previous preferences were moved to "core" while these ones are moving to "core/editor". I need to think how better to address this.

Reply:
Ok, so I'm just going to update everything to use the "core" scope. instead of "core/editor"